### PR TITLE
[zh-cn] sync controllers/deployment.md

### PR DESCRIPTION
**Pull request by my-git9**
content/zh-cn/docs/concepts/workloads/controllers/deployment.md

### CODE REVIEW

**Review by windsonsea — APPROVED** [review]
/lgtm

File: content/zh-cn/docs/concepts/workloads/controllers/deployment.md [hunk 1]
```diff
@@ -2144,11 +2144,112 @@ For example, when this value is set to 30%, the new ReplicaSet can be scaled up
...
+Here are some Rolling Update Deployment examples that use the `maxUnavailable` and `maxSurge`:
 -->
 例如，当此值为 30% 时，启动滚动更新后，会立即对新的 ReplicaSet 扩容，同时保证新旧 Pod
 的总数不超过所需 Pod 总数的 130%。一旦旧 Pod 被杀死，新的 ReplicaSet 可以进一步扩容，
 同时确保更新期间的任何时候运行中的 Pod 总数最多为所需 Pod 总数的 130%。
 
+以下是一些使用 `maxUnavailable` 和 `maxSurge` 的滚动更新 Deployment 的示例：
+
+{{< tabs name="tab_with_md" >}}
+{{% tab name="Max Unavailable" %}}
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/workloads/controllers/deployment.md:2157]: should we translate the name tag?

File: content/zh-cn/docs/concepts/workloads/controllers/deployment.md [hunk 2]
```diff
@@ -2144,11 +2144,112 @@ For example, when this value is set to 30%, the new ReplicaSet can be scaled up
...
+        ports:
+        - containerPort: 80
+  strategy:
+    type: RollingUpdate
+    rollingUpdate:
+      maxUnavailable: 1
+ ```
+
+{{% /tab %}}
+{{% tab name="Max Surge" %}}
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/workloads/controllers/deployment.md:2188]: this one also

File: content/zh-cn/docs/concepts/workloads/controllers/deployment.md [hunk 3]
```diff
@@ -2144,11 +2144,112 @@ For example, when this value is set to 30%, the new ReplicaSet can be scaled up
...
+        ports:
+        - containerPort: 80
+  strategy:
+    type: RollingUpdate
+    rollingUpdate:
+      maxSurge: 1
+ ```
+
+{{% /tab %}}
+{{% tab name="Hybrid" %}}
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/workloads/controllers/deployment.md:2219]: and here

**Pull request merged**